ubuntu/derived/standard/Dockerfile.template

```diff
@@ -55,5 +69,9 @@ ENV ANT_HOME=/usr/share/ant \
     dotnet=/usr/bin/dotnet \
     GRADLE_HOME=/usr/share/gradle \
     grunt=/usr/local/bin/grunt \
-    JAVA_HOME=/usr/lib/jvm/java-$(OPENJDK_VERSION)-openjdk-amd64 \
     M2_HOME=/usr/share/maven \
+    JAVA_HOME_6_X64=/usr/lib/jvm/java-6-oracle \
```

Comment:
Sorry to be picky, but can we keep this list of environment variables in alphabetical order? Thanks!